### ISSUE TITLE
Design | Profile | Accessibility | Mobile #

### ISSUE THREAD

**Issue by SKasimow** [department-of-veterans-affairs/va.gov-team#49589]
## Background

This ticket is to determine best design options before proceeding with FE work.The Accessibility team noted the following issue for screen reader users and the need to have content properly labeled:

> Within Profile Notification Settings, near the top of the content, the Veteran’s mobile number is listed, followed by a link to update their mobile phone number. The text content listed beforehand does not truly indicate what type of phone number is being read aloud by screen readers. Users may be able to guess that this is their mobile number, but the lack of specific callout text may create confusion.

## Tasks
- [x] Create design(s) to address this issue and review with Accessibility team
- [x] Document feedback
- [x] Adjust designs as necessary
- [x] Review with PM and PO
  
## Acceptance Criteria
- [ ] Designs are approved and ready for FE work
- [ ] Link to sketch file is provided to PM

## Associated Tickets
- #46609

**Comment by andaleliz** [comment]
I reviewed this with the accessibility folks, and the recommendation is to:
- Add a label in front of the piece of contact information to say what it is. We'll use a bold label in front of the piece of contact information. **Mobile phone** for mobile phones, and **email address** for email addresses.

- The links can be changed to say "update", but in an aria-label we should be more specific and say "mobile number" or "email address" for screenreaders on the link.

The sketch files have been updated to reflect this new format - devs can look at any screens on the [deskop](https://www.sketch.com/s/afd69a1f-72d2-430b-9b62-285e9d3f479c) or [mobile pages](https://www.sketch.com/s/afd69a1f-72d2-430b-9b62-285e9d3f479c/p/B5792AF8-87F2-496D-AE7A-DAD1448A6018/canvas) of this prototype.

@SKasimow and @Samara-Strauss how does this sound to you? I can close it out if you're good with these and FE can take it from here.

**Comment by Samara-Strauss** [comment]
All of this sounds good to me!

**Comment by SKasimow** [comment]
Me too! 

@ericasauv09 - could you open a FE ticket for this change (lmk if you have any questions)...thanks!

**Comment by ericasauv09** [comment]
@SKasimow I created #49885
